flocker_bb/builders/flocker_vagrant.py

```diff
@@ -497,8 +499,10 @@ def getBuilders(slavenames):
                       category='flocker',
                       factory=buildTutorialBox(),
                       nextSlave=idleSlave),
-        BuilderConfig(name='flocker/installed-package/fedora-20',
-                      builddir='flocker-installed-package-fedora-20',
+        BuilderConfig(name='flocker/installed-package/' +
+                      TUTORIAL_DISTRIBUTION,
+                      builddir='flocker-installed-package-' +
+                      TUTORIAL_DISTRIBUTION,
                       slavenames=slavenames['fedora-20/vagrant'],
```

Comment:
Tom explained that the "fedora-20" here is describing the OS the slave is running.  Since we only use this to run a VM of some other OS with Vagrant it shouldn't matter (and changing it involves setting up brand new slaves on a new OS which bleechh).